virtualenv_embedded/site.py

```diff
@@ -738,7 +738,9 @@ def write_history():
     def import_builtin_stuff():
         """PyPy specific: pre-import some built-in modules to match CPython."""
         import encodings
-        import exceptions
+
+        if "exceptions" in sys.builtin_module_names:
```

Comment:
would be nice to refer to some kind of pypy documentation/link about why this is needed